pkg/apps/util/scheme.go

```diff
@@ -24,20 +23,12 @@ var (
 func init() {
 	utilruntime.Must(appsv1.Install(annotationDecodingScheme))
 	utilruntime.Must(appsv1.DeprecatedInstallWithoutGroup(annotationDecodingScheme))
-	// TODO eventually we shouldn't deal in internal versions, but for now decode into one.
```

Comment:
@deads2k this is supposed to be the "external" version of utils and I guess we don't want to import the internal apis here?

Reply:
This is only needed for DecodeDeploymentConfig method that decodes the annotation where DC is stored. AFAIK the DC is either:

1) oapi (legacy) DC (v1/DeploymentConfig)
2) grouped DC

We need to deal with both, but I'm not sure we need to decode to "internal", but rather switch to use external in all helpers that care. This will be a follow up.